### ISSUE TITLE
could you add the path in the graphraft.py files?llm1_adapter_dir, llm2_model_dir,

### ISSUE THREAD

**Issue by ToneLi** [AlfredClemedtson/graphraft#2]
Hi,  
  could you add the path in the graphraft.py files?llm1_adapter_dir, llm2_model_dir,
 parser.add_argument("--dataset", type=str, required=True)
    parser.add_argument("--llm1_model_dir", type=str, default="neo4j/text2cypher-gemma-2-9b-it-finetuned-2024v1" )
    parser.add_argument("--llm1_adapter_dir", type=str, default=None)
    parser.add_argument("--llm1_beam_width", type=int, default=5)
    parser.add_argument("--llm2_model_dir", type=str, default=None)
    parser.add_argument("--llm2_max_sequence_length", type=int, default=11_000)
    parser.add_argument("--add_more_answers", action="store_true")

**Comment by AlfredClemedtson** [comment]
Hi @ToneLi, thanks for the suggestion! Since we don't provide the trained models and you have to train them yourself (see README), the path paths depends on what you name them. Checkout `llm1.py` and `llm2.py` to see where the models are saved if you've already trained them. Does this answer you question?

**Comment by ToneLi** [comment]
<img width="1629" height="750" alt="Image" src="https://github.com/user-attachments/assets/e31fd623-fd54-4856-bba7-c1307590940b" />

<img width="1827" height="735" alt="Image" src="https://github.com/user-attachments/assets/e7f9abf1-4990-4d2d-97d8-f4f10d76fdba" />  thanks,  how could we solve this issues?

**Comment by ToneLi** [comment]
Sorry for bothering you again. Could you help organize the code more clearly? It will make it much easier for us to reproduce your results.

**Comment by AlfredClemedtson** [comment]
Did you create the vector indexes? [https://neo4j.com/docs/cypher-manual/current/indexes/semantic-indexes/vector-indexes/](https://neo4j.com/docs/cypher-manual/current/indexes/semantic-indexes/vector-indexes/). 

For MAG I think we use one index per label/node type, which is used for the NER and then an additional index for the paper abstracts used by the retriever. Keep in mind that each index is linked to a label, if you need an index covering multiple node types (like for Prime) you need a common base label.
